client/ReviewModule.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import $ from 'jquery';
 import TopBar from './TopBar.jsx';
 import Reviews from './Reviews.jsx';
+// import ReviewStyles from './styles/ReviewStyles';
```

Comment:
Do you need this? if not, then can remove.

Reply:
Fixed.